src/patternfly/components/Label/styles.scss

```diff
@@ -0,0 +1,46 @@
+@import "../../patternfly-utilities";
+
+.pf-c-label {
+  // Component
+  --pf-c-label--PaddingTop: var(--pf-global--spacer--xs);
+  --pf-c-label--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-label--PaddingBottom: var(--pf-global--spacer--xs);
+  --pf-c-label--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-label--LineHeight: var(--pf-global--LineHeight--md);
+  --pf-c-label--BorderRadius: var(--pf-global--BorderRadius--sm);
+
+  // Color
+  --pf-c-label--BackgroundColor: var(--pf-global--primary-color--100);
+  --pf-c-label--Color: var(--pf-global--Color--light-100);
+
+  // Font
+  --pf-c-label--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-label--FontWeight: var(--pf-global--FontWeight--regular);
```

Comment:
you can remove this and the `font-weight` property. fyi, there is no `--pf-global--FontWeight--regular`

Reply:
ah just realized its "normal", so we dont need to specify it because the default is normal?

Reply:
Yep! assuming the font weight for labels should be the same as regular text, no need to specify it here.